docker/checkout.bash

```diff
@@ -41,10 +41,10 @@ if [[ "$BRANCH" = "develop" ]]; then
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch $BRANCH
       git clone --depth=1 https://github.com/usdot-fhwa-stol/novatel_gps_driver.git --branch $BRANCH
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch $BRANCH
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch $BRANCH
+      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch release/Wanderer
```

Comment:
Can you revert to $BRANCH instead of hardcoding the branch name? We have automated builds set up to put the right branch there at build time.